src/org/gk/render/RenderableChemicalDrug.java

```diff
@@ -1,16 +1,14 @@
 package org.gk.render;
 
-import java.awt.Color;
-
 /**
  * This customized Renderer is for ChemicalDrug.
  * @author wug
  *
  */
-public class RenderableChemicalDrug extends RenderableChemical {
+public class RenderableChemicalDrug extends RenderableChemical implements DefaultRenderConstants {
```

Comment:
Don't add implements here. This has been taken care in supclass.

Reply:
Understood.

Reply:
Fixed in [e77f0a9](https://github.com/reactome/CuratorTool/pull/23/commits/e77f0a99bd4c6699021ff050f23d7c451e9e7d35).